### DISCUSSION TITLE
1.4.0 - another huge update

### DISCUSSION THREAD

**Discussion by hawkeye116477** [hawkeye116477/playnite-legendary-plugin#44] [Announcements]
* Added possibility to upload playtime to cloud (experimental feature)
* Added cloud settings to per game settings (manual sync, etc)
* Added possibility to move games to other base path
* Added DLC Manager for installed games (so non-launchable DLCs won't be longer added to library, instead they will show up here)
* Added question if user wants to switch offline mode for playing game if can't sign in
* Added possibility to display download speed in bits per second
* Added possibility to check all extra content in Game Installer
* Added offline mode as possible feature to metadata provider
* Improved games updating (added optional automatic updates and now DLCs will be updated together with base game)
* Added possibility to display message when new launcher version will appear
* Added possibility to remove game settings while uninstalling game
* Added possibility to activate Ubisoft and EA titles (Plugin settings=>Additional services)
* Improved clearing cache
* Improved Game Installer appearance
* Moved EOS Overlay settings to new Additional services tab
* Renamed game menu item 'Legendary launcher settings' to 'Modify launch settings'
* Playtime synchronization is now disabled by default
* Ubisoft Connect games won't be imported to library now
* All DLCs shouldn't start downloading now if user don't want them
* Disabled few things from metadata provider, cuz they can be not proper
* Removed parsing ini config file (that was used only to set some values as default)
* Fixed detecting installed games for Heroic 2.9.0+
* Fixed few Download Manager bugs
* Improved error handling
* Added Portuguese (Brazilian) translation by BrutuZ

<hr /><em>This discussion was created from the release <a href='https://github.com/hawkeye116477/playnite-legendary-plugin/releases/tag/1.4.0'>1.4.0</a>.</em>

**Comment by BrutuZ** [comment]
> - Ubisoft Connect games won't be imported to library now

Could this be a setting? I don't use any Ubisoft integration on Playnite since I buy all their games elsewhere and more often than not it requires both clients to run

**Reply by hawkeye116477** [reply]
I'll think about that. However in case of Legendary after activating games from settings, they require only one launcher - Ubisoft Connect (https://github.com/derrod/legendary/blob/7fefdc497329276fc8c41d20e72600e0112352d1/legendary/cli.py#L257).

**Comment by BrutuZ** [comment]
Can't say I am, only played Ubi games with Steam so far. But the point is that without an Ubisoft WhateverTheyCallTheirServiceThisYear integration\plugin I won't see those games at all in my Playnite library.

**Reply by hawkeye116477** [reply]
I added that now => https://github.com/hawkeye116477/playnite-legendary-plugin/commit/ffb7fd2f7490b666f2a2e5f8c9d0387c62bf7dab, but it will stay as disabled by default. Looks like problem also might be that **some games like Watch Dogs 2 can't be installed by Legendary, so you'll still need Ubisoft plugin**.

There may be also problem with tracking playtime of R6 Siege (https://github.com/JosefNemec/PlayniteExtensions/commit/4f8ebec110087c2154ff277179e2bf866d8d0985) if it can be installed with Legendary, I don't have it, so don't know its id, so it will need to stay like that for now unless someone will report it or send PR :smile:.

**Reply by BrutuZ** [reply]
Good enough IMO. I just want to be able to see them in the library so they're considered when browsing what to try next, I'll worry about _how to make it work_ (going after an integration \ client) later xD
